src/main/java/org/aerogear/digger/client/DiggerClient.java

```diff
@@ -250,7 +251,8 @@ public JobWithDetails getJob(String name) throws DiggerClientException {
      */
     public BuildTriggerStatus build(String jobName, long timeout, Map<String, String> params) throws DiggerClientException {
         try {
```

Comment:
maybe we should rename this to sth like `buildAndWait` (I am terrible at naming things)

Reply:
@aliok I advised this to remain the same as I didn't want to break the existing api, wdyt? More appropriate maybe to concentrate on the naming of the new interfaces?

Reply:
ok, makes sense @PhilipGough @cfoskin